README.md

```diff
@@ -61,13 +72,13 @@ $ docker logs hazelcast-quarkus1 -f
 2019-12-13 23:40:27,299 INFO  [io.quarkus] (main) Installed features: [cdi, resteasy, resteasy-jackson, resteasy-jsonb]...
 ```
 
-Send a put operation to the first microservice running on port 8081:
+Send a put operation to the first application running on port 8081:
 ```
 $ curl -X POST "localhost:8081/hazelcast/put?key=key_1&value=value_1";echo;
 {"containerName":"hazelcast-quarkus_2","value":"value_1"}
 ```
 
-Get the value from microservice running on 8080 and verify that it is the same value as put operation:
+Get the value from application running on 8080 and verify that it's the same value as put operation:
 ```
 $ curl "localhost:8080/hazelcast/get?key=key_1";echo;
 {"containerName":"hazelcast-quarkus_1","value":"value_1"}
```

Comment:
Do you think we should rename `hazelcast-quarkus_1` too? quarkus_replica_1 or microservice_replica_1 could be a better name.

Reply:
I don't really have a preference, I will update in the next PR